js/client/modules/@arangodb/testsuites/export.js

```diff
@@ -121,82 +121,64 @@ function exportTest (options) {
   databases.forEach((name, idx) => {
     try {
       fs.removeDirectory(tmpPath);
-    } catch (err) {}
-    
-    const args = {
+    } catch (err) {
+    }
+
+    const commonArgValues = {
       'configuration': fs.join(pu.CONFIG_DIR, 'arangoexport.conf'),
       'server.username': options.username,
       'server.password': options.password,
       'server.endpoint': instanceInfo.endpoint,
       'server.database': name,
-      'collection': 'UnitTestsExport',
       'type': 'json',
```

Comment:
It would be good if we didn't set the `type` globally, but rather set it individually for each test case.